src/main/java/com/github/krzysiekjodlowski/ox/OxConsoleGame.java

```diff
@@ -0,0 +1,110 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.board.Board;
+import com.github.krzysiekjodlowski.ox.board.BoardValidator;
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+import com.github.krzysiekjodlowski.ox.ui.UI;
+
+/**
+ * Main loop for OX game.
+ * ATTENTION - For now it runs only one
+ * "round" and without win condition check!
+ *
+ * @author Krzysztof Jodlowski
+ */
+class OxConsoleGame implements Game {
+  private final UI<String, Integer> ui;
+  private final EventBus oxGameEventBus = new OxGameEventBus();
+  private Players players = new Players.Builder().build();
+
+  OxConsoleGame(UI<String, Integer> ui) {
+    this.ui = ui;
+  }
+
+  /**
+   * ATTENTION - Below implementation is temporary!
+   * Analogical one will appear in future Round class.
+   * Also all hardcoded Strings will be read from file by Settings.
+   */
+  @Override
+  public void run() {
+    this.welcomePlayers();
+    Board board = this.setBoardSideLength();
+    BoardValidator boardValidator = new BoardValidator(board, this.setWinCondition());
+    this.setStartingPlayer();
+    this.oxGameEventBus.register(board);
+    this.oxGameEventBus.register(boardValidator);
+    this.oxGameEventBus.register(this.players);
```

Comment:
Nice to see few event bus subscribers :)

Reply:
And there will be more!